src/pybind/mgr/cephadm/serve.py

```diff
@@ -695,8 +696,8 @@ def _apply_service(self, spec: ServiceSpec) -> bool:
                 public_networks = [x.strip() for x in out.split(',')]
                 self.log.debug('mon public_network(s) is %s' % public_networks)
 
-        def matches_network(host):
-            # type: (str) -> bool
+        def matches_public_network(host, sspec):
+            # type: (str, ServiceSpec) -> bool
```

Comment:
As you advised me the other day, if you're touching the function signature, convert it to the full type annotations form.